src/packages/ganache/README.md

```diff
@@ -381,8 +381,7 @@ Fork:
 
                                         Use the shorthand command ganache --fork to automatically fork from
                                         Mainnet at the latest block.
-                                        [choices: "mainnet", "ropsten", "kovan", "rinkeby", "goerli", "görli",
-                                                                                                    "sepolia"]
+                                        [choices: "mainnet", "ropsten", "rinkeby", "goerli", "görli", "sepolia"]
```

Comment:
I think this line is one character longer than it's supposed to be. Can you go back to indenting it?